src/apb/engine.py

```diff
@@ -208,9 +248,25 @@ def cmdrun_init(**kwargs):
     bindable = kwargs['bindable']
     async = kwargs['async']
     params = kwargs['params']
+    skip = kwargs['skip'] or []
 
     description = "This is a sample application generated by apb init"
 
+    # Ensure that command line options are acceptable
+    for skip_option in skip:
```

Comment:
@rthallisey THANK YOU! I knew there was a way to do this with argparse and it was driving me nuts I couldn't remember.